src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -7143,6 +7172,52 @@ def _update_artifact_type(params):
     logger.warning("Deployment type: %s. To override deployment type, please specify the --type parameter. "
                    "Possible values: war, jar, ear, zip, startup, script, static", params.artifact_type)
 
+def _get_instances_list_arm_internal(params):
+    url = _build_get_instances_list_arm_url(params)
+    response = send_raw_request(params.cmd.cli_ctx, "GET", url)
+    if response.status_code == 200:
+        logger.info("Instances list retrieved successfully.")
+        return response.json().get("value", [])
+    raise CLIError("Unable to get instances list.")
+
+def _get_instance_id_internal(param):
+    instances = _get_instances_list_arm_internal(param)
+
+    if not instances or len(instances) == 0:
+        raise CLIError("No instances found.")
+    
+    logger.info("returning instance id for instance: %s", instances[0]['name'])
+    return instances[0]['name']
+
+
+def _get_affinity_cookie_internal(param):
+    instance = _get_instance_id_internal(param)
+    logger.info("returning affinity cookie for instance: %s", instance)
```

Comment:
is this going to be visible to cx in their console?

Reply:
in azcli logger.info is only visible in debug or verbose mode 
the regular logs you see in console are logger.warning logs

Reply:
@Shi1810 let's format any debug logs you want to add. I see missing capitalization for all such logs